accelerator/errors.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #define SC_MODULE_CONF (0x07 << SC_MODULE_SHIFT)
 #define SC_MODULE_UTILS (0x08 << SC_MODULE_SHIFT)
 #define SC_MODULE_HTTP (0x09 << SC_MODULE_SHIFT)
+#define SC_MODULE_STORAGE (0x10 << SC_MODULE_SHIFT)
```

Comment:
Why didn't you use `0xA` instead of `0x10`?

Reply:
It should be 0xA.